storagenode/storagenodedb/infodb.go

```diff
@@ -23,7 +25,20 @@ var ErrInfo = errs.Class("infodb")
 
 // InfoDB implements information database for piecestore.
 type InfoDB struct {
-	db *sql.DB
+	db        *sql.DB
+	space     *space
+	bandwidth *bw
+}
+
+type space struct {
+	usedSpace int64
+	once      *sync.Once
```

Comment:
the concurrency protection primitives go above fields it protects, see https://github.com/storj/storj/wiki/Style#mutexes

Reply:
This doesn't need to be a pointer

Reply:
updated